modules/stanford_affiliate_organization_views/stanford_affiliate_organization_views.views_default.inc

```diff
@@ -273,6 +275,203 @@ function stanford_affiliate_organization_views_views_default_views() {
+  $handler->display->display_options['relationships']['field_s_affiliate_type_tid']['id'] = 'field_s_affiliate_type_tid';
+  $handler->display->display_options['relationships']['field_s_affiliate_type_tid']['table'] = 'field_data_field_s_affiliate_type';
+  $handler->display->display_options['relationships']['field_s_affiliate_type_tid']['field'] = 'field_s_affiliate_type_tid';
+  $handler->display->display_options['defaults']['fields'] = FALSE;
+  /* Field: Content: Link */
+  $handler->display->display_options['fields']['field_s_affiliate_link']['id'] = 'field_s_affiliate_link';
+  $handler->display->display_options['fields']['field_s_affiliate_link']['table'] = 'field_data_field_s_affiliate_link';
+  $handler->display->display_options['fields']['field_s_affiliate_link']['field'] = 'field_s_affiliate_link';
+  $handler->display->display_options['fields']['field_s_affiliate_link']['label'] = '';
+  $handler->display->display_options['fields']['field_s_affiliate_link']['exclude'] = TRUE;
+  $handler->display->display_options['fields']['field_s_affiliate_link']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['field_s_affiliate_link']['click_sort_column'] = 'url';
+  $handler->display->display_options['fields']['field_s_affiliate_link']['type'] = 'link_plain';
+  /* Field: Content: Image */
+  $handler->display->display_options['fields']['field_s_affiliate_image']['id'] = 'field_s_affiliate_image';
+  $handler->display->display_options['fields']['field_s_affiliate_image']['table'] = 'field_data_field_s_affiliate_image';
+  $handler->display->display_options['fields']['field_s_affiliate_image']['field'] = 'field_s_affiliate_image';
+  $handler->display->display_options['fields']['field_s_affiliate_image']['label'] = '';
+  $handler->display->display_options['fields']['field_s_affiliate_image']['alter']['make_link'] = TRUE;
+  $handler->display->display_options['fields']['field_s_affiliate_image']['alter']['path'] = '[field_s_affiliate_link]';
+  $handler->display->display_options['fields']['field_s_affiliate_image']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['field_s_affiliate_image']['click_sort_column'] = 'fid';
+  $handler->display->display_options['fields']['field_s_affiliate_image']['settings'] = array(
+    'image_style' => 'sm-profile-scaled',
+    'image_link' => '',
+  );
+  /* Field: Content: Title */
+  $handler->display->display_options['fields']['title']['id'] = 'title';
+  $handler->display->display_options['fields']['title']['table'] = 'node';
+  $handler->display->display_options['fields']['title']['field'] = 'title';
+  $handler->display->display_options['fields']['title']['label'] = '';
+  $handler->display->display_options['fields']['title']['alter']['alter_text'] = TRUE;
+  $handler->display->display_options['fields']['title']['alter']['text'] = '<a href="[field_s_affiliate_link]">[title]</a>';
```

Comment:
I did try using the settings for the field first and it was ignored when grouping. [&#X02639]

Joe was the one who showed me the rewrite.


________________________________
From: Sherakama <notifications@github.com>
Sent: Wednesday, February 17, 2016 9:13 AM
To: SU-SWS/stanford_affiliate_organization
Cc: Caryl J Westerberg
Subject: Re: [stanford_affiliate_organization] Soe 583 (#6)


In modules/stanford_affiliate_organization_views/stanford_affiliate_organization_views.views_default.inc<https://github.com/SU-SWS/stanford_affiliate_organization/pull/6#discussion_r53197320>:

> +  $handler->display->display_options['fields']['field_s_affiliate_image']['label'] = '';
> +  $handler->display->display_options['fields']['field_s_affiliate_image']['alter']['make_link'] = TRUE;
> +  $handler->display->display_options['fields']['field_s_affiliate_image']['alter']['path'] = '[field_s_affiliate_link]';
> +  $handler->display->display_options['fields']['field_s_affiliate_image']['element_label_colon'] = FALSE;
> +  $handler->display->display_options['fields']['field_s_affiliate_image']['click_sort_column'] = 'fid';
> +  $handler->display->display_options['fields']['field_s_affiliate_image']['settings'] = array(
> +    'image_style' => 'sm-profile-scaled',
> +    'image_link' => '',
> +  );
> +  /* Field: Content: Title */
> +  $handler->display->display_options['fields']['title']['id'] = 'title';
> +  $handler->display->display_options['fields']['title']['table'] = 'node';
> +  $handler->display->display_options['fields']['title']['field'] = 'title';
> +  $handler->display->display_options['fields']['title']['label'] = '';
> +  $handler->display->display_options['fields']['title']['alter']['alter_text'] = TRUE;
> +  $handler->display->display_options['fields']['title']['alter']['text'] = '<a href="[field_s_affiliate_link]">[title]</a>';


@cjwest<https://github.com/cjwest>, I'm pretty sure you can set this field to be linked as a setting instead of wrapping it in an tag. I'm pretty sure the default output of links comes with more than just the href property as well.

-
Reply to this email directly or view it on GitHub<https://github.com/SU-SWS/stanford_affiliate_organization/pull/6/files#r53197320>.